packages/docs/src/routes/docs/(qwik)/components/events/index.mdx

```diff
@@ -243,9 +241,9 @@ export default component$(() => {
 
 ## Custom event props
 
-When creating your components it is often useful to pass custom event props that look like event handlers, (even though they are not DOM events, only callbacks). Component boundaries in Qwik must be serializable for the optimizer to split them up into separate chunks, and functions are not serializable unless they are converted to a QRL using the `$` sign.
+When creating your components it is often useful to pass custom event props that look like event handlers, (even though they are not DOM events, only callbacks). Component boundaries in Qwik must be serializable for the optimizer to split them up into separate chunks, and functions are not serializable unless they are converted to a QRL.
```

Comment:
Do we state somewhere that doing `<Cmp foo$={() => "bar"}/>` will be automatically changed into `<Cmp foo$={$(() => "bar")}/>` by the optimizer? (but not `<Cmp foo$={"bar"}/>`)

Reply:
I'm not sure I'm following.

I removed 

"using the `$` sign" piece, because you could type `fn$: () => void` and the eslint plugin will still complain.

Reply:
When I read, "using the `$` sign", I understand that `fn$: () => void` will work but this isn't the right way to do it and the eslint plugin will complain.

I think removing that makes it clearer that what the optimizer needs is "code that's converted to a QRL", whether it is by using `$()` in the components or `QRL<>` at the type level.

I removed it because it bugged me when I read it. Like the entire section is about using QRL<>, so if it's written that writing '$' at the end of a word is enough it can be quite confusing.

Btw, QRL<> is a good api because it expresses the right type in the form of types. The '$' sign at the prop type level would mean that we add QRL (or PropFnInterface) behind the scenes for the developer. I think it's best that the developer is aware of the QRL<> type directly.